compiler-core/src/build/package_compiler.rs

```diff
@@ -75,6 +82,7 @@ where
             target,
             write_metadata: true,
             perform_codegen: true,
+            skip_analysis: false,
```

Comment:
This approach means that it still loads caches and emits warnings. How about instead we don't load any modules at all? That's what I did when I implemented this. I wish I knew what happened to my version of this 😅